Chore: Dependencies update december

McNamara84 opened the pull request:
Updated dependencies for (PHP and JS) for this month. Maybe the last update this year.

## Changes & Rationale
- `composer update`
- `npm update`

## Related Issues
None.

## Definition of Done Checklist

- [ ] User story requirements met
- [ ] Documentation updated
- [ ] New code covered by unit tests
- [x] All unit tests pass locally
- [x] Manually tested
- [x] Works on desktop devices
- [ ] Works on mobile devices

## Laravel-Specific Changes

- [ ] Migrations
- [ ] Models
- [ ] Views
- [ ] Routes
- [ ] Seeders
- [ ] Config
- [ ] Artisan commands

## Infrastructure & Tooling Changes

- [x] PHP dependencies
- [x] Node.js dependencies
- [ ] Tooling/scripts
- [ ] CI/CD workflows
- [ ] Repository meta/templates

## Additional Notes
None.

McNamara84 committed:
Update Composer dependencies

Updated multiple PHP dependencies in composer.lock to their latest versions, including laravel/framework, symfony components, league packages, and others. Also removed the phiki/phiki package. This ensures compatibility with the latest features, bug fixes, and security updates.

McNamara84 committed:
Update dependencies in package-lock.json

Upgraded esbuild and related platform packages from 0.25.10 to 0.25.12, rollup platform packages from 4.52.0 to 4.53.3, and axios from 1.12.2 to 1.13.2 to include the latest features and security updates.

McNamara84 committed:
Downgrade several Symfony packages to v7.4.x

Reverts symfony/clock, symfony/css-selector, symfony/event-dispatcher, symfony/string, and symfony/translation from v8.x to v7.4.x in composer.lock. Also adds a platform override for PHP 8.2.0.

McNamara84 committed:
Add license and PHP platform version to composer.json

Added the MIT license and specified PHP 8.2.0 as the required platform version in composer.json for better clarity and compatibility management.

nicolaibach commented on composer.json:
```diff
@@ -2,6 +2,7 @@
     "$schema": "https://getcomposer.org/schema.json",
     "name": "mcnamara84/ladis",
     "type": "project",
+    "license": "MIT",
```
license should be `GPL-3.0-only` or `GPL-3.0-or-later`?

McNamara84 committed:
Merge branch 'main' into chore/dependencies-update-december

McNamara84 committed:
Update project license to GPL-3.0-or-later

Changed the license in composer.json from MIT to GPL-3.0-or-later to reflect a new licensing choice for the project.

McNamara84 committed:
Update psy/psysh to v0.12.17 in composer.lock

McNamara84 committed:
Update rollup and related dependencies to 4.53.4

Bump rollup and all platform-specific rollup packages from version 4.53.3 to 4.53.4 in package-lock.json to include the latest fixes and improvements.

McNamara84 committed:
Update maintainer contact info in composer.json

Changed Holger Ehrmann's email and homepage to reflect updated contact details.

McNamara84 commented:
However that could have happened...
I took the opportunity to update my email address and homepage as well.

nicolaibach reviewed (approved):
Thanks!
